src/pages/Bridge/IssueForm/index.tsx

```diff
@@ -270,28 +269,11 @@ const IssueForm = (): JSX.Element | null => {
     };
 
     const onSubmit = async (data: IssueFormData) => {
-      // Set status first as there will be a delay while fetching vaults
-      setSubmitStatus(STATUSES.PENDING);
-      const btcAmount = BitcoinAmount.from.BTC(data[BTC_AMOUNT]);
-
-      // Get vaults on submit to ensure data is current
-      // Check vault capacity
-      const vaultCapacityError = await validateVaultCapacity(btcAmount);
-
-      // Manually append error to input field
-      if (vaultCapacityError) {
-        setError(BTC_AMOUNT, {
-          type: 'manual',
-          message: vaultCapacityError
-        });
-
-        setSubmitStatus(STATUSES.RESOLVED);
-
-        return;
-      }
-
       try {
-        const wrappedTokenAmount = BitcoinAmount.from.BTC(data[BTC_AMOUNT]);
+        setSubmitStatus(STATUSES.PENDING);
+        await requestLimitsRefetch();
+        await trigger(BTC_AMOUNT);
```

Comment:
This is the main update for resolving this issue.

Before submitting, we refetch the vault capacity data by invalidating the query in case the data is stale by the time the user makes the submission.
And we trigger the validation based on newly fetched data.